test/sub_transport/test_basic.py

```diff
@@ -292,6 +292,17 @@ async def test_init(self):
         """Test null modem init"""
         SerialTransport(asyncio.get_running_loop(), mock.Mock(), "dummy")
 
+    async def test_loop(self):
```

Comment:
We do not test that the libraries work correctly, so I see this test as not needed.

Reply:
I added this test more as illustration that something is wrong in the original code. I will remove it.

Please check again the production code change. Is the "protocol" event loop necessarily always the same as the one of the transport? Anyhow, the original access was faulty.